automation/prow_periodic_push.sh

```diff
@@ -2,9 +2,24 @@
 set -e
 build_date="$(date +%Y%m%d)"
 cat "$QUAY_PASSWORD" | docker login --username $(cat "$QUAY_USER") --password-stdin=true quay.io
-export DOCKER_TAG="${build_date}_$(git show -s --format=%h)-arm64"
+
+case $BUILD_ARCH in
+  crossbuild-s390x|s390x)
+    ARCH_SUFFIX="-s390x"
+    ;;
+  crossbuild-aarch64|aarch64)
+    ARCH_SUFFIX="-arm64"
+    ;;
+  *)
+    echo "No BUILD_ARCH or ${BUILD_ARCH}, assuming amd64"
+    ;;
+esac
+
+export DOCKER_TAG="${build_date}_$(git show -s --format=%h)${ARCH_SUFFIX}"
+
 make manifests
 make bazel-push-images
+
 bucket_dir="kubevirt-prow/devel/nightly/release/kubevirt/containerized-data-importer/${build_date}"
-gsutil cp ./_out/manifests/release/cdi-operator.yaml gs://$bucket_dir/cdi-operator-arm64.yaml
-gsutil cp ./_out/manifests/release/cdi-cr.yaml gs://$bucket_dir/cdi-cr-arm64.yaml
+gsutil cp ./_out/manifests/release/cdi-operator.yaml gs://$bucket_dir/cdi-operator${ARCH_SUFFIX}.yaml
+gsutil cp ./_out/manifests/release/cdi-cr.yaml gs://$bucket_dir/cdi-cr${ARCH_SUFFIX}.yaml
```

Comment:
missing the hyphen? or intended?

Reply:
oh, my bad, it's up above

Reply:
intended, as the SUFFIX includes the hyphen, to prevent repetition
so that with no `BUILD_ARCH` specified there is no extra hyphen left